main.py

```diff
@@ -6,13 +6,18 @@
 from led_controller import LEDController
 from current_spotify_playback import CurrentSpotifyPlayback, NoArtworkException
 from spotify_background_color import SpotifyBackgroundColor
+import rpi_ws281x as neopixel
 
 
 CLIENT_ID = os.environ.get('SPOTIPY_CLIENT_ID')
 CLIENT_SECRET = os.environ.get('SPOTIPY_CLIENT_SECRET')
 REDIRECT_URI = os.environ.get('SPOTIPY_REDIRECT_URI')
 REFRESH_TOKEN = os.environ.get('SPOTIPY_REFRESH_TOKEN')
 
+def fillWithColor(strip, r, g ,b):
+  for i in range(strip.numPixels()):
+    strip.setPixelColorRGB(i, r, g, b)
+  strip.show()
```

Comment:
I would like to see this in an equivalent class to `LEDController` but for WS281X or to create a more general class that includes both types of led strips.